packages/client/components/PrimaryButton.tsx

```diff
@@ -6,11 +6,12 @@ import {Radius} from 'types/constEnums'
 import {Elevation} from 'styles/elevation'
 
 const StyledBaseButton = styled(BaseButton)((props: BaseButtonProps) => {
-  const {disabled, waiting} = props
+  const {disabled, waiting, buttonName} = props
   const visuallyDisabled = disabled || waiting
   return {
     backgroundImage: visuallyDisabled ? PALETTE.GRADIENT_WARM_LIGHTENED : PALETTE.GRADIENT_WARM,
     borderRadius: Radius.BUTTON_PILL,
+    buttonName,
```

Comment:
can remove, these are just for styles. emotion will pass all props through to the wrapped element

Reply:
Ok I'll remove that.